Makefile

```diff
@@ -1,6 +1,8 @@
 BASE_PATH = .
 include Makefile-constants.mk
 
+NPROCS ?= 6
```

Comment:
How about doing something like this instead?
```make
NPROCS ?= $(shell nproc)
```

Reply:
I agree, I'll give it a try !